snow/README.md

```diff
@@ -16,7 +16,9 @@ graph LR
 
 ## Intro
 
-The Avalanche network consists of 3 built-in blockchains: X-Chain, C-Chain, and P-Chain. The X-Chain is used to manage assets and uses the Avalanche consensus protocol. The C-Chain is used to create and interact with smart contracts and uses the Snowman consensus protocol. The P-Chain is used to coordinate validators and stake and also uses the Snowman consensus protocol. At the time of writing, the Avalanche network has ~1200 validators. A set of validators makes up a subnet. Subnets can validate 1 or more chains. It is a common misconception that 1 subnet = 1 chain and this is shown by the primary subnet of Avalanche which is made up of the X-Chain, C-Chain, and P-Chain.
+The Avalanche primary network consists of 3 built-in blockchains: the X-Chain, C-Chain, and P-Chain. All three chains rely on the Snowman consensus protocol. The X-Chain, which previously used DAG-based Avalanche consensus, was upgraded to Snowman in the Cortina network update. 
+
+The X-Chain is used to manage assets. The C-Chain is used to create and interact with smart contracts. The P-Chain is used to coordinate validators and stake. At the time of writing, the Avalanche network has ~1200 validators. A set of validators makes up a subnet. Subnets can validate 1 or more chains. It is a common misconception that 1 subnet = 1 chain and this is shown by the primary subnet of Avalanche which is made up of the X-Chain, C-Chain, and P-Chain.
```

Comment:
Should we keep a structure more similar to the original for the X-chain? Something like
`All three chains rely on the Snowman consensus protocol.
The X-Chain is used to manage assets; it previously used DAG-based Avalanche consensus and was upgraded to Snowman in the Cortina network update.`

Reply:
Thanks @abi87. I'm not sure I can go either way. I can't merge regardless, so let's see how others feel.

Reply:
My impression of this doc is the "current state of affairs" - for example, we don't mention Snowman++ here